pkg/executor/aggfuncs/builder.go

```diff
@@ -457,7 +457,7 @@ func buildGroupConcat(ctx sessionctx.Context, aggFuncDesc *aggregation.AggFuncDe
 	default:
 		// The last arg is promised to be a not-null string constant, so the error can be ignored.
 		c, _ := aggFuncDesc.Args[len(aggFuncDesc.Args)-1].(*expression.Constant)
-		sep, _, err := c.EvalString(nil, chunk.Row{})
+		sep, _, err := c.EvalString(ctx, chunk.Row{})
```

Comment:
Make sure `EvalString` not panic. But I don't know why the previous code used `nil` as the input parameter. @XuHuaiyu PTAL